staging/src/k8s.io/client-go/dynamic/fake/simple.go

```diff
@@ -145,23 +192,65 @@ func (c *dynamicResourceClient) UpdateStatus(obj *unstructured.Unstructured) (*u
 	return ret, err
 }
 
-func (c *dynamicResourceClient) Delete(name string, opts *metav1.DeleteOptions) error {
-	_, err := c.client.Fake.
-		Invokes(testing.NewDeleteAction(c.resource, c.namespace, name), &metav1.Status{Status: "dynamic delete fail"})
+func (c *dynamicResourceClient) Delete(name string, opts *metav1.DeleteOptions, subresources ...string) error {
```

Comment:
out of curiosity: do we have subresource deletion anywhere?

Reply:
> out of curiosity: do we have subresource deletion anywhere?

it's logically possible.  nothing immediately comes to mind though.